styles/index.css

```diff
@@ -1,3 +1,75 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-content {
```

Comment:
The overlay text can be positioned by overlay-container. This is unneccessary.
Also the background image can be defined by the container

Reply:
Yes I removed the overlay-content, and added background-image to container